src/tests/multihost/alltests/test_sssctl_analyzer.py

```diff
@@ -175,9 +175,99 @@ def test_analyzer_pam_logs(self, multihost, backupsssdconf):
             pytest.fail(f"{user} failed to login")
         else:
             client.logout()
-        _, stdout = analyzer(multihost, 'show 1 --pam')
+        _, stdout = analyze(multihost, 'show 1 --pam')
         assert 'CID #1' in stdout
         pam_cmds = ['SSS_PAM_AUTHENTICATE', 'SSS_PAM_AUTHENTICATE',
                     'SSS_PAM_ACCT_MGMT', 'SSS_PAM_SETCRED']
         for pam_auth in pam_cmds:
             assert pam_auth in stdout
+
+    def test_analyze_tevent_id(self, multihost, backupsssdconf):
+        """
+        :title: sssctl analyze to parse tevent chain IDs from logs
+        :id: f748766c-0177-4306-9e7f-816586734e14
+        :description: sssctl analyze should able to parse tevent chain
+         IDs from responder logs
+        :bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=2013259
+        :steps:
+        1. Configure sssd to authenticate against directory server
+        2. Enable debug_level to 9 in the 'nss', 'pam' and domain section
+        3. Restart SSSD with cleared cache
+        4. Log in as a user using ssh
+        5. Confirm tevent chain IDs(RID) is showing in logs
+        :expectedresults:
+        1. Should succeed
+        2. Should succeed
+        3. Should succeed
+        4. Should succeed
+        5. Should succeed
+        """
+        tools = sssdTools(multihost.client[0])
+        multihost.client[0].service_sssd('stop')
+        tools.remove_sss_cache('/var/lib/sss/db/')
+        tools.remove_sss_cache('/var/log/sssd/')
+        dm_sec = ['nss', 'pam']
+        sssd_params = {'debug_level': '9'}
+        for sec_op in dm_sec:
+            tools.sssd_conf(sec_op, sssd_params, action='update')
+        multihost.client[0].service_sssd('start')
+        i_cmd = f'id foo1@{ds_instance_name}'
+        multihost.client[0].run_command(i_cmd, raiseonerr=False)
+        user = f'foo1@{ds_instance_name}'
+        client_hostname = multihost.client[0].sys_hostname
+        client = pexpect_ssh(client_hostname, user, 'Secret123',
+                             debug=False)
+        try:
+            client.login()
+        except SSHLoginException:
+            pytest.fail(f"{user} failed to login")
+        else:
+            client.logout()
+        _, stdout = analyze(multihost, 'show 1')
+        assert 'RID#' in stdout
```

Comment:
It could be useful to assert the username matches as well

Reply:
Added username assertion.